src/re_frame/interop.clj

```diff
@@ -0,0 +1,43 @@
+(ns re-frame.interop)
+
+(defn next-tick [f]
+  (.start (Thread. f)))
```

Comment:
Actually, `future` is not the right answer (but nor is `(.start (Thread. f))`.

Given that the purpose of this function is for `re-frame.router/event-queue` to manage the dispatch of events, and given that CLJS is single-threaded, it would be kind of silly to have the tests multi-threading the processing of dispatched events.  I'll change this to use a single-thread executor.